### PR TITLE
refactor: move Stanza to Dune_lang

### PR DESCRIPTION
**Pull request by rgrinberg**
Clearing away more unrelated stuff from Build_engine

### CODE REVIEW

**Comment by rgrinberg** [comment]
@snowleopard ping on this simple one. Should be fairly simple as I'm just moving stuff around.

File: src/dune_engine/action_ast.ml [hunk 1]
```diff
@@ -1,10 +1,7 @@
 open! Stdune
 open Import
 open Dune_lang.Decoder
-
-let () =
-  let module M = Stanza in
-  ()
```
Comment (snowleopard) [inline src/dune_engine/action_ast.ml:7]: What was this about?

Comment (rgrinberg) [reply src/dune_engine/action_ast.ml:7]: Probably to force ocamldep's behavior and introduce a dependecy on stanza.ml. I recall we had to do ugly hacks like that the old bootstrap procedure.

**Review by snowleopard — APPROVED** [review]
Looks good!

File: src/dune_engine/action_ast.ml (hunk 1 above)
Comment (snowleopard) [reply src/dune_engine/action_ast.ml:7]: Interesting. Thanks!

**Pull request merged**